DEBUG - 2025-04-03 00:00:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:00:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:01:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:01:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:05:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:12:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:15:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-03 00:19:01 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: PUT, Route: pedido/1]
in SYSTEMPATH\BaseModel.php on line 1010.
 1 SYSTEMPATH\BaseModel.php(1010): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Pedido.php(150): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedido->update('1')
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedido))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 00:19:35 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: PUT, Route: pedido/2]
in SYSTEMPATH\BaseModel.php on line 1010.
 1 SYSTEMPATH\BaseModel.php(1010): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Pedido.php(150): CodeIgniter\Model->update('2', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedido->update('2')
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedido))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
DEBUG - 2025-04-03 00:22:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:22:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:22:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:23:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:23:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 00:23:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-03 00:35:40 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: producto]
in SYSTEMPATH\BaseModel.php on line 825.
 1 SYSTEMPATH\BaseModel.php(825): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\ProductoController.php(29): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->create()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 00:36:26 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: producto]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\ProductoController.php(28): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->create()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 00:37:37 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: producto]
in SYSTEMPATH\BaseModel.php on line 825.
 1 SYSTEMPATH\BaseModel.php(825): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([], true)
 3 APPPATH\Controllers\ProductoController.php(29): CodeIgniter\Model->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->create()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
DEBUG - 2025-04-03 01:56:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 01:58:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:00:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:00:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:03:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:04:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:05:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:12:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:15:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:16:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:17:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:17:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:17:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:18:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:19:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:20:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:20:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:20:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:20:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:22:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 02:23:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-03 02:48:39 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: register]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\AuthController.php(16): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 02:49:13 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: register]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\AuthController.php(16): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 02:50:17 --> CodeIgniter\HTTP\Exceptions\HTTPException: Failed to parse JSON string. Error: Syntax error
[Method: POST, Route: register]
in SYSTEMPATH\HTTP\IncomingRequest.php on line 537.
 1 SYSTEMPATH\HTTP\IncomingRequest.php(537): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidJSON('Syntax error')
 2 APPPATH\Controllers\AuthController.php(16): CodeIgniter\HTTP\IncomingRequest->getJSON(true)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 4 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
DEBUG - 2025-04-03 02:51:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:32:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:34:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:34:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-03 04:34:06 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: POST, Route: producto]
in SYSTEMPATH\BaseModel.php on line 1830.
 1 SYSTEMPATH\BaseModel.php(1830): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\BaseModel.php(805): CodeIgniter\BaseModel->transformDataToArray([], 'insert')
 3 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([], true)
 4 APPPATH\Controllers\ProductoController.php(55): CodeIgniter\Model->insert([])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->create()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
DEBUG - 2025-04-03 04:40:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:42:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:48:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:51:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:53:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 04:58:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:02:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:03:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:04:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-03 05:05:22 --> CodeIgniter\HTTP\Exceptions\BadRequestException: The URI you submitted has disallowed characters: "producto_67ee16e3c7f898.71220099.jpeg""
[Method: GET, Route: uploads/uploads/productos/producto_67ee16e3c7f898.71220099.jpeg%22]
in SYSTEMPATH\Router\Router.php on line 739.
 1 SYSTEMPATH\Router\Router.php(207): CodeIgniter\Router\Router->checkDisallowedChars('uploads/uploads/productos/producto_67ee16e3c7f898.71220099.jpeg"')
 2 SYSTEMPATH\CodeIgniter.php(832): CodeIgniter\Router\Router->handle('uploads/uploads/productos/producto_67ee16e3c7f898.71220099.jpeg"')
 3 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
DEBUG - 2025-04-03 05:12:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:12:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:12:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:12:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:12:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:13:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:13:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:13:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:13:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 05:30:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 06:20:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 06:55:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 07:04:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 07:06:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 07:53:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-03 07:53:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-03 08:17:43 --> CodeIgniter\Filters\Exceptions\FilterException: "App\Filters\Cors" filter must have a matching alias defined.
[Method: GET, Route: producto]
in SYSTEMPATH\Filters\Filters.php on line 842.
 1 SYSTEMPATH\Filters\Filters.php(842): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('App\\Filters\\Cors')
 2 SYSTEMPATH\Filters\Filters.php(487): CodeIgniter\Filters\Filters->processAliasesToClass('before')
 3 SYSTEMPATH\Filters\Filters.php(218): CodeIgniter\Filters\Filters->initialize('producto')
 4 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('producto', 'before')
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:17:43 --> CodeIgniter\Filters\Exceptions\FilterException: "App\Filters\Cors" filter must have a matching alias defined.
[Method: GET, Route: producto]
in SYSTEMPATH\Filters\Filters.php on line 842.
 1 SYSTEMPATH\Filters\Filters.php(842): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('App\\Filters\\Cors')
 2 SYSTEMPATH\Filters\Filters.php(487): CodeIgniter\Filters\Filters->processAliasesToClass('before')
 3 SYSTEMPATH\Filters\Filters.php(218): CodeIgniter\Filters\Filters->initialize('producto')
 4 SYSTEMPATH\CodeIgniter.php(479): CodeIgniter\Filters\Filters->run('producto', 'before')
 5 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
ERROR - 2025-04-03 08:21:47 --> mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo' in C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\app\Controllers\AuthController.php(30): CodeIgniter\Model->insert(Array)
#8 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->register()
#9 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\Users\\diana\\...')
#15 {main}
CRITICAL - 2025-04-03 08:21:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:21:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$JBlBVf7rN5pDhv.GV8HMd.ZbPoGPkc3xQ/rsf7zvd4NiCNsRDRYRC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:47\', \'2025-04-03 08:21:47\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$JBlBVf7rN5pDhv.GV8HMd.ZbPoGPkc3xQ/rsf7zvd4NiCNsRDRYRC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:47\', \'2025-04-03 08:21:47\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:21:47 --> [Caused by] mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$JBlBVf7rN5pDhv.GV8HMd.ZbPoGPkc3xQ/rsf7zvd4NiCNsRDRYRC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:47\', \'2025-04-03 08:21:47\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$JBlBVf7rN5pDhv.GV8HMd.ZbPoGPkc3xQ/rsf7zvd4NiCNsRDRYRC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:47\', \'2025-04-03 08:21:47\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$JBlBVf7rN5pDhv.GV8HMd.ZbPoGPkc3xQ/rsf7zvd4NiCNsRDRYRC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:47\', \'2025-04-03 08:21:47\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
ERROR - 2025-04-03 08:21:58 --> mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo' in C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\app\Controllers\AuthController.php(30): CodeIgniter\Model->insert(Array)
#8 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->register()
#9 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\Users\\diana\\...')
#15 {main}
CRITICAL - 2025-04-03 08:21:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:21:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$9kSD3HuH5K6UbN6S3tOnF.18SgmVV.OM53qeeAEm5X520crTwPq/i\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:58\', \'2025-04-03 08:21:58\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$9kSD3HuH5K6UbN6S3tOnF.18SgmVV.OM53qeeAEm5X520crTwPq/i\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:58\', \'2025-04-03 08:21:58\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:21:58 --> [Caused by] mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$9kSD3HuH5K6UbN6S3tOnF.18SgmVV.OM53qeeAEm5X520crTwPq/i\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:58\', \'2025-04-03 08:21:58\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$9kSD3HuH5K6UbN6S3tOnF.18SgmVV.OM53qeeAEm5X520crTwPq/i\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:58\', \'2025-04-03 08:21:58\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$9kSD3HuH5K6UbN6S3tOnF.18SgmVV.OM53qeeAEm5X520crTwPq/i\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:58\', \'2025-04-03 08:21:58\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
ERROR - 2025-04-03 08:21:59 --> mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo' in C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\app\Controllers\AuthController.php(30): CodeIgniter\Model->insert(Array)
#8 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->register()
#9 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\Users\\diana\\...')
#15 {main}
CRITICAL - 2025-04-03 08:21:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:21:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$Q6cA60FacIth5C3o1AEV/e/iWzhBkJe4p/ugcE9/yrMEm.pgl02aC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:59\', \'2025-04-03 08:21:59\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$Q6cA60FacIth5C3o1AEV/e/iWzhBkJe4p/ugcE9/yrMEm.pgl02aC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:59\', \'2025-04-03 08:21:59\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:21:59 --> [Caused by] mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$Q6cA60FacIth5C3o1AEV/e/iWzhBkJe4p/ugcE9/yrMEm.pgl02aC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:59\', \'2025-04-03 08:21:59\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$Q6cA60FacIth5C3o1AEV/e/iWzhBkJe4p/ugcE9/yrMEm.pgl02aC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:59\', \'2025-04-03 08:21:59\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$Q6cA60FacIth5C3o1AEV/e/iWzhBkJe4p/ugcE9/yrMEm.pgl02aC\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:21:59\', \'2025-04-03 08:21:59\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
ERROR - 2025-04-03 08:22:01 --> mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo' in C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\app\Controllers\AuthController.php(30): CodeIgniter\Model->insert(Array)
#8 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\AuthController->register()
#9 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\diana\Documents\xampp\htdocs\ecommerce-B2C-backend\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\Users\\diana\\...')
#15 {main}
CRITICAL - 2025-04-03 08:22:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:22:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$W9P6GsOAWNNs/ddv41jrWO9b52N2WzJ4D.3gjApEq1dnuFByf5ldO\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:22:01\', \'2025-04-03 08:22:01\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$W9P6GsOAWNNs/ddv41jrWO9b52N2WzJ4D.3gjApEq1dnuFByf5ldO\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:22:01\', \'2025-04-03 08:22:01\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
CRITICAL - 2025-04-03 08:22:01 --> [Caused by] mysqli_sql_exception: Duplicate entry 'diana@test.do' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$W9P6GsOAWNNs/ddv41jrWO9b52N2WzJ4D.3gjApEq1dnuFByf5ldO\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:22:01\', \'2025-04-03 08:22:01\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$W9P6GsOAWNNs/ddv41jrWO9b52N2WzJ4D.3gjApEq1dnuFByf5ldO\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:22:01\', \'2025-04-03 08:22:01\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (\'Diana\', \'Bello\', \'2023-03-02\', \'diana@test.do\', \'$2y$10$W9P6GsOAWNNs/ddv41jrWO9b52N2WzJ4D.3gjApEq1dnuFByf5ldO\', \'8294687156\', \'SHOPPER\', \'2025-04-03 08:22:01\', \'2025-04-03 08:22:01\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `fecha_nacimiento`, `correo`, `contrasena`, `telefono`, `rol`, `created_at`, `updated_at`) VALUES (:nombre:, :apellido:, :fecha_nacimiento:, :correo:, :contrasena:, :telefono:, :rol:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(30): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->register()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\diana\\Documents\\xampp\\htdocs\\ecommerce-B2C-backend\\public\\index.php')
